src/App.js

```diff
@@ -1,30 +1,40 @@
-import React, { useState, useEffect } from 'react';
-import { Routes, Route } from 'react-router-dom';
-import { getAllData } from './util/index';
-import { HomePage } from './components/HomePage/HomePage';
-import { LoginPage } from './components/LoginPage/LoginPage';
-import ProtectedRoute from './components/ProtectedRoute/ProtectedRoute';
+import React, { useState, useEffect } from "react";
+import { getAllData } from "./util/index";
 
-const URL = 'http://localhost:8000/api/v1/';
```

Comment:
This still needs to be included. Please add it back in.

Reply:
This was only deleted on the PR, I still have it on my local branch. Sorry about that!